src/app/search/query-builder.service.ts

```diff
@@ -44,13 +45,20 @@ export class QueryBuilderService {
         bucketStubs: any, filters: any, sortModeMap: any): string {
         const count = this.getNumberOfFilters(filters);
         let sidebarBody = bodybuilder().size(query_size);
+        sidebarBody = this.excludeContent(sidebarBody);
         sidebarBody = this.appendQuery(sidebarBody, values, advancedSearchObject, searchTerm);
         sidebarBody = this.appendAggregations(count, sidebarBody, bucketStubs, filters, sortModeMap);
         const builtSideBarBody = sidebarBody.build();
         const sideBarQuery = JSON.stringify(builtSideBarBody);
         return sideBarQuery;
     }
 
+    private excludeContent(body: any) {
+      // TODO: it should be possible to exclude tags and workflowVersions too
+      // however, it currently breaks the contents of the datatables for some reason
+      return body.rawOption('_source', {'excludes': ['*.content', '*.sourceFiles', 'description']});
```

Comment:
Try [_sourceExclude](https://github.com/danpaz/bodybuilder#other-options) for bodybuilder

Reply:
Excluding workflowVersions causes datatables to break because it's using the workflowVersion to check whether the tool/workflow is verified or not

Reply:
Oh, seems like next version we need to equivalent of a sql `group by`

Reply:
Could we exclude the attributes for versions except for the verified field?

Reply:
Yup, as a temporary fix